Data/Services/CustomerService.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Data.Entities;
+using Data.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Data.Repositories
+{
+    public class CustomerService : ICustomerService
+    {
+        private readonly List<Customer> _customers = new();
+
+        public void Create(Customer createdCustomer)
+        {
+            bool emailAlreadyExists = _customers.Any(customer => customer.Email == createdCustomer.Email);
```

Comment:
```suggestion
            var emailAlreadyExists = _customers.Any(customer => customer.Email == createdCustomer.Email);
```
O retorno será um bool, não há necessidade de definir um tipo para a variável.